var/spack/repos/builtin/packages/mxnet/package.py

```diff
@@ -128,4 +129,8 @@ def install(self, spec, prefix):
         # install python bindings
         if '+python' in spec:
             python = which('python')
-            python('python/setup.py', 'install', '--prefix={0}'.format(prefix))
+            # The python libs are in a separate dir, and it is necessary to change
+            # directory so that setup.py picks them up.
+            os.chdir('python')
+            python('setup.py', 'install', '--prefix={0}'.format(prefix),
+                   '--single-version-externally-managed', '--root=/')
```

Comment:
```suggestion
            with working_dir('python'):
                python('setup.py', 'install', '--prefix={0}'.format(prefix),
                       '--single-version-externally-managed', '--root=/')
```

Reply:
That was fast :) Agreed that it is better.